src/main/rx/expresslrs_impl.h

```diff
@@ -0,0 +1,110 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// expresslrs packet header types
+// 00 -> standard 4 channel data packet
+// 01 -> switch data packet
+// 11 -> tlm packet
+// 10 -> sync packet with hop data
+typedef enum {
+    ELRS_RC_DATA_PACKET = 0x00,
+    ELRS_MSP_DATA_PACKET = 0x01,
+    ELRS_SYNC_PACKET = 0x02,
+    ELRS_TLM_PACKET = 0x03,
+} elrs_packet_type_e;
+
+typedef enum {
+    DIO_UNKNOWN = 0,
+    DIO_RX_DONE = 1,
+    DIO_TX_DONE = 2
+} dioReason_e;
+
+typedef enum {
+    CONNECTED,
```

Comment:
Would be good to have a common prefix on this enum's definitions for clarity as in those above.

Reply:
`ELRS_` prefix added